frontend_v2/src/app/components/home/home.component.html

```diff
@@ -429,8 +429,30 @@ <h3 class="fw-light center">Sponsors</h3>
         </div>
       </div>
     </section>
-    <!-- Subscribe to newsletter -->
     <section class="ev-container text-med-black ev-super-light-bg">
```

Comment:
Css seems a bit distorted for mobile devices. Can you please try making `padding:0px` for `.grad-container` just for the testimonial container.

Reply:
The left and right buttons? Because I had seen the mobile view as well. I will check it once again

Reply:
@Kajol-Kumari Currently, the mobile view is: 
![Screenshot from 2021-07-14 00-36-00](https://user-images.githubusercontent.com/24366008/125511643-061f94c1-7ca7-4be6-8fbe-e31a736c61b0.png)

Reply:
IMO we can use the left and right padding spaces just for mobile devices. It would look much cleaner that way.

Reply:
Got it. Done the required changes.
![Screenshot from 2021-07-14 18-09-53](https://user-images.githubusercontent.com/24366008/125623687-d72198e9-1992-4386-814b-49f9643b441b.png)